docs/migration-recipe.md

```diff
@@ -0,0 +1,117 @@
+Migrating to new host:
+----------------------
+
+### Prepare salt configuration for migration:  
+
+1.  Ensure that salt-master, loadbalancer, and host in question are up, and that their health check for the relevant service is failing in haproxy after the host is fully up
+
+To view haproxy status: 
+
+-vagrant up the salt-master, loadbalancer, and host in question (`vagrant up`)
+
+-Prepare an ssh configuration file to access the host with native ssh commands: `vagrant ssh-config salt-master loadbalancer >> vagrant-ssh` 
+
+-Open an ssh session with port forwarding to the haproxy status page: `ssh -L 4646:127:0.0.1:464 -F vagrant-ssh loadbalancer`
+
+-view the haproxy status page in your browser `loacalhost:4646/haproxy?stats` 
+
+2.  Edit pillar data for roles.sls to include both old and new hostnames (ex. hostname*)
+
+### Migrate the host:
+
+#### Update Salt Master with latest config including prep from above
+
+1.  ssh into the salt-master server `ssh salt-master`
+
+2.  Navigate to srv/psf-salt `user@salt:~$ cd srv/psf-salt`
+
+3.  Run `user@salt:/srv/psf-salt$ sudo git pull`
+
+4.  Run highstate to update the roles settings to reflect the new matchng pattern, as well as additional changes to support migration: `user@salt:/srv/psf-salt$ sudo salt-call state.highstate`
+
+#### Ensure new configuration doesn't impact host being migrated
+
+1.  ssh into the old host `ssh hostname`
```

Comment:
Let's make sure we're keeping consistent with the example host names, so I think this line might want to be:

"1.  ssh into the old-host `ssh old-host`"